README.md

```diff
@@ -172,6 +172,20 @@ The projects in the [`deployment-platforms`](./deployment-platforms) directory s
 - [Indie Stack](https://github.com/remix-run/indie-stack): Remix Stack for deploying to Fly with SQLite, authentication, testing, linting, formatting
 - [Blues Stack](https://github.com/remix-run/blues-stack): Remix Stack for deploying to Fly with PostgreSQL, authentication, testing, linting, formatting
 
+## Badges
+
+[![Made with Prisma](http://made-with.prisma.io/dark.svg)](https://prisma.io) [![Made with Prisma](http://made-with.prisma.io/indigo.svg)](https://prisma.io)
+
+Built something awesome with Prisma? 🌟 Show it off with these [badges](https://github.com/prisma/presskit?tab=readme-ov-file#badges), perfect for your readme or website.
+
+```
```

Comment:
we could use a markdown annotation for syntax highlighting here:

````suggestion
```md
````

Reply:
Thanks for the suggestion! I've tried this on presskit and just didn't see any difference 😄 Also expected syntax highlighting. We can add this later if needed! 👌